recipes/river-ingester/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x -e
+set -o pipefail
+
+if [[ "${target_platform}" == osx-* ]]; then
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+    CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cd cpp
+mkdir -p build/release
+cd build/release
+cmake -G "Unix Makefiles" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DRIVER_BUILD_INGESTER=ON \
+  -DRIVER_BUILD_TESTS=OFF \
+  -DRIVER_INSTALL=ON \
+  ${CMAKE_ARGS} \
+  ../..
+cd ingester
+make
```

Comment:
Is there a `make check` or `ctest` that we can run here?

Reply:
Unfortunately the tests currently expect a particular testing host setup (i.e. Redis running) and thus are typically run manually in the source repo . A task for the future is making our tests a bit smarter to be able to run unit tests and only the integration tests requiring Redis if available.

That said, there's at least a check later in the meta.yaml that the `river-ingester` binary is built and in the appropriate place.